packages/client/src/actions/actions.test.ts

```diff
@@ -218,25 +220,28 @@ describe("actions", () => {
     ], { $returnEdits: true });
 
     expect(result).toMatchInlineSnapshot(` 
-    {
-  "addedLinksCount": 0,
-  "addedObjectCount": 0,
-  "deletedLinksCount": 0,
-  "deletedObjectsCount": 0,
-  "edits": [
-    {
-      "objectType": "Office",
-      "primaryKey": "SEA",
-      "type": "modifyObject",
-    },
-    {
-      "objectType": "Office",
-      "primaryKey": "NYC",
-      "type": "modifyObject",
-    },
-  ],
-  "modifiedObjectsCount": 2,
-  "type": "edits",
-}`);
+      {
+        "addedLinks": [],
+        "addedObjects": [],
+        "deletedLinksCount": 0,
+        "deletedObjectsCount": 0,
+        "editedObjectTypes": [
+          "Office",
+        ],
+        "modifiedObjects": [
+          {
+            "objectType": "Office",
+            "primaryKey": "SEA",
+            "type": "modifyObject",
+          },
+          {
+            "objectType": "Office",
+            "primaryKey": "NYC",
+            "type": "modifyObject",
+          },
+        ],
+        "type": "edits",
+      }
+  `);
   });
 });
```

Comment:
FLUP: Might be good to write some tests/stubs that can handle the return of largeScaleEdits to flesh out the tests

Reply:
Good idea!